src/pretalx/settings.py

```diff
@@ -187,7 +187,11 @@
         },
     },
     'loggers': {
-        '': {'handlers': ['file', 'console'], 'level': loglevel, 'propagate': True},
+        '': {
```

Comment:
What's the reason for this change?

Reply:
readability / more consistend with the other entries.
sorry. i will not do this again...